# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1262301384 28800
#      Thu Dec 31 15:16:24 2009 -0800
# Node ID f91d41387a050452e167b1fec18651ecb7818884
# Parent  2d7a8c227713df8c90b3a5c72be71b5c4ef648ea
Reverse using move_if_change rule for cruft.def (Changeset:9980)

Rule interferes with make algorithm for determining when files are up-to-date

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-31  Rik <octave@nomad.inbox5.com>
+
+	* Makefile.am: Reverse using simple_move_if_change_rule for cruft.def
+	which interferes with make timestamp algorithm.
+
 2009-12-19  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Eliminate TOPDIR variable in favor of built-in automake
 	variables of top_builddir and top_srcdir.
 
 2009-12-13  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Use simple_move_if_change_rule for cruft.def to eliminate
diff --git a/libcruft/Makefile.am b/libcruft/Makefile.am
--- a/libcruft/Makefile.am
+++ b/libcruft/Makefile.am
@@ -54,14 +54,14 @@ include quadpack/module.mk
 include ranlib/module.mk
 include slatec-err/module.mk
 include slatec-fn/module.mk
 include villad/module.mk
 
 cruft.def: $(libcruft_la_SOURCES) mkf77def
 	chmod a+rx mkf77def
 	./mkf77def $(srcdir) $(libcruft_la_SOURCES) > $@-t
-	$(simple_move_if_change_rule)
+	mv $@-t $@
 
 EXTRA_DIST += ChangeLog STOP.patch mkf77def.in
 
 DISTCLEANFILES = cruft.def
 
